pyscript.core/src/stdlib/pyweb/pydom.py

```diff
@@ -1,9 +1,32 @@
 import sys
-import warnings
-from functools import cached_property
-from typing import Any
 
-from pyodide.ffi import JsProxy
+try:
+    import warnings
+except ImportError:
+    # TODO: For now it probably means we are in MicroPython. We should figure
+    # out the "right" way to handle this. For now we just ignore the warning
+    # and logging to console
+    class warnings:
+        @staticmethod
+        def warn(*args, **kwargs):
+            print("WARNING: ", *args, **kwargs)
+
+    warnings = None
+
+try:
+    from functools import cached_property
+except ImportError:
+    # TODO: same comment about micropython as above
+    cached_property = property
+
+try:
+    from pyodide.ffi import JsProxy
+except ImportError:
+    # TODO: same comment about micropython as above
+    def JsProxy(obj):
```

Comment:
my only concern here is *instance* brand check where it would always be `False` but maybe that's OK/desired ... if that's the case, this is fine, as long as `isinstance(obj, JsProxy)` won't break the world if executed.

Reply:
mmm... good point. I think we can make the fix "safer" but I'd think we are overall safe enough to assume it won't be an issue. There are 2 reasons for this assessment:

1. It's only an issue in MicroPython and `JSProxy` don't exist in MP anyway
2. It'd be an issue if we were exposing the whole `pydom` module but we are not, what you get when you import `from pyweb import pydom` is an instance of `PyDom` (the class) and not the module,  so you don't really have access to the `JSProxy` mock.